tests/e2e/configurer/base.go

```diff
@@ -52,18 +52,41 @@ func (bc *baseConfigurer) GetChainConfig(chainIndex int) *chain.Config {
 }
 
 func (bc *baseConfigurer) RunValidators() error {
+	errChan := make(chan error, len(bc.chainConfigs))
+
+	// Launch goroutines for each chainConfig
 	for _, chainConfig := range bc.chainConfigs {
-		if err := bc.runValidators(chainConfig); err != nil {
+		go func(config *chain.Config) {
+			err := bc.runValidators(config)
+			errChan <- err
+		}(chainConfig)
+	}
+
+	// Collect errors from goroutines
+	for range bc.chainConfigs {
+		if err := <-errChan; err != nil {
 			return err
 		}
 	}
+
```

Comment:
We should do this for only one of `RunValidators` and `runValidators` right? Right now were making more threads than needed / its a bit confusing

Reply:
Before this change:
- All A chain spins up at once
- All B chain spins up at once

With this change:
- All A and B chains spin up at once

Maybe a 5 second speedup so I can remove if you disagree

Reply:
Oh I see, that makes sense